src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/RecordARecallDecisionResult.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.remandandsentencing.model.Recall
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.validation.ValidationMessage
+import java.time.LocalDate
+import java.util.UUID
+
+data class RecordARecallDecisionResult(
+  val decision: RecordARecallDecision,
+  val validationMessages: List<ValidationMessage> = emptyList(),
+  val recallableSentences: List<RecallableSentence> = emptyList(),
+  val eligibleRecallTypes: List<Recall.RecallType> = emptyList(),
+)
+
+data class RecallableSentence(
+  val sentenceSequence: Int,
+  val bookingId: Long,
+  val uuid: UUID,
+  val sentenceCalculation: RecallSentenceCalculation,
+)
+
+data class RecallSentenceCalculation(
+  val calculatedRelease: LocalDate,
```

Comment:
I'm unsure what sentenceGroupRelease means. May benefit from comments or swagger doc explaining these 3 fields are

The one I'm really unsure of is `sentenceGroupRelease`